storybookjs/storybook
# Controls: Fix file controls not resetting

https://github.com/storybookjs/storybook/issues/19921

## What I did

Manually set `value` of file input to `null` when control `value` is `null` to make `Reset controls` works with file input controls

## How to test

- Run yarn storybook:ui in code dir
- Navigate to `http://localhost:6006/?path=/story/storybook-blocks-controls-files--accept-anything`
- Verify that resetting works on file control
- Demo : https://www.loom.com/share/1fa2ce8920364b4cb3b21b1efb283d5d
